corehq/apps/geospatial/templates/case_grouping_map.html

```diff
@@ -34,7 +34,7 @@
   <div class="row">
     {% include 'geospatial/partials/saved_polygon_filter.html' %}
   </div>
-  <div class="alert alert-info" data-bind="visible: shouldRefreshPage">
+  <div class="alert alert-info" style="margin-top:10px;" data-bind="visible: shouldRefreshPage">
```

Comment:
Nit: Does it work when you use the `mt-x` class instead where x is a number? If possible, this would be nicer than changing the style directly.

Reply:
I tried earlier but was not able to get it working. I think it is because these are not available in Bootstrap 3 which is what currently used in geospatial app.